docs/pages/platform_guides/environments.mdx

```diff
@@ -0,0 +1,115 @@
+
+export const meta = {
+  title: "Configure environments",
+  __filename
+};
+
+export default ({ children }) => <Documentation {...meta}>{children}</Documentation>;
+
+## Introduction
+
+Environments can be used to configure different variations of your schedules.
+
+A common use-case for environments is to run a staged release process; you might
+want to push your code to a `staging` environment first, and then push it to a `production`
+environment once the code has been tested.
+
+## Configuring environments
+
+Environments are configured in your Dataform project's `environments.json` file.
+
+An environment consists of:
+
+- a name
+- a Git reference at which the code should be run (either a branch name or a commit SHA)
+- (optionally) overridden values for the project's configuration (the settings in `dataform.json`)
+
+A simple example of an `environments.json` file is:
+
+```json
+{
+  "environments": [
+    {
+      "name": "staging",
+      "gitReference": {
+        "branch": "master"
+      },
+      "configOverride": {
+        "defaultSchema": "dataform_staging",
+        "assertionSchema": "dataform_assertions_staging"
+      }
+    }
+  ]
+}
+```
+
+This `staging` environment runs the code (and schedules) on the project's `master` branch. It also overrides the values of
+`defaultSchema` and `assertionSchema` to isolate `staging` schedule runs from those in other environments.
+
+Note that Dataform uses the `environments.json` file on your `master` branch to determine your project's environments.
+Any changes to your environments must be pushed to `master` before Dataform will take note of those changes.
+
+<div className="bp3-callout bp3-icon-info-sign bp3-intent-warning" markdown="1">
+  If your project has a missing or empty <code>environments.json</code> file, Dataform uses a
+  default environment which runs the code on your <code>master</code> branch with no project
+  configuration overrides.
+</div>
+
+## Multiple environments
+
+Once your team or project outgrows the default environment configuration, a typical configuration might look like the following:
+
+```json
+{
+  "environments": [
+    {
+      "name": "staging",
+      "gitReference": {
+        "branch": "master"
+      },
+      "configOverride": {
+        "defaultSchema": "dataform_staging",
+        "assertionSchema": "dataform_assertions_staging"
+      }
+    },
+    {
+      "name": "production",
+      "gitReference": {
+        "branch": "production"
+      },
+      "configOverride": {
+        "defaultSchema": "dataform_prod",
+        "assertionSchema": "dataform_assertions_prod"
+      }
+    }
+  ]
+}
+```
+
+This configuration creates a `staging` environment which always runs the latest version of your project's code
+(as exists on the `master` branch). When you have confirmed that the code works as expected, you can push that to production
+by merging the `master` branch into the `production` branch.
+
```

Comment:
I think here would be a good place to explain how this is going to impact schedules and that schedules will run for every environments.

Reply:
I've changed the copy in the Intro section and the Multiple environments section to hopefully make this clearer, lmk!